projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -23,20 +25,24 @@ import {TUI_SANITIZER} from '@taiga-ui/core';
     host: {
         class: 'tui-editor-socket',
     },
+    providers: [TuiDestroyService],
 })
 export class TuiEditorSocketComponent {
     @Input()
+    set editor(editor: AbstractTuiControl<string>) {
+        editor.control?.valueChanges
+            .pipe(takeUntil(this.destroy$))
```

Comment:
`this.destroy$` will kill subscription on destroy of the component. It is nice.

But what happens if Input-property changes twice (or more) during the component's life?
Will the component have only single subscription ?